core/scheduler/src/main/resources/application.conf

```diff
@@ -37,10 +38,11 @@ akka {
     }
   }
 
-  remote.netty.tcp {
-    send-buffer-size = 3151796b
-    receive-buffer-size = 3151796b
-    maximum-frame-size = 3151796b
+  remote {
+    artery {
```

Comment:
As the Akka version is upgraded, it is now using artery rather than netty which is deprecated.
https://doc.akka.io/docs/akka/current/remoting.html#classic-remoting-deprecated-